fe2-android/app/src/main/res/layout/fragment_launch.xml

```diff
@@ -1,78 +1,76 @@
-                    android:gravity="center"
-                    android:text="@string/text_title_launch"
-                    android:textSize="@dimen/size_title"
-                    android:textStyle="bold"/>
+      <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:text="@string/text_title_launch"
+        android:textSize="@dimen/size_title"
+        android:textStyle="bold" />
 
-            <TextView
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:gravity="center"
-                    android:text="@string/text_body_launch"
-                    android:textSize="@dimen/size_body"
-                    android:textStyle="italic"/>
+      <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:text="@string/text_body_launch"
+        android:textSize="@dimen/size_body"
+        android:textStyle="italic" />
 
-            <EditText
-                    android:id="@+id/entry_box_launch"
-                    android:afterTextChanged="@{(edtitable)->view_model.setLaoName(edtitable.toString())}"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginHorizontal="@dimen/margin_text"
-                    android:autofillHints=""
-                    android:hint="@string/text_entry_launch"
-                    android:inputType="text"/>
+      <EditText
+        android:id="@+id/entry_box_launch"
+        android:afterTextChanged="@{(edtitable)->view_model.setLaoName(edtitable.toString())}"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/margin_text"
+        android:autofillHints=""
+        android:hint="@string/text_entry_launch"
+        android:inputType="text" />
 
-            <LinearLayout
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginVertical="@dimen/margin_button"
-                    android:gravity="center_horizontal"
-                    android:orientation="horizontal">
+      <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginVertical="@dimen/margin_button"
+        android:gravity="center_horizontal"
+        android:orientation="horizontal">
 
-                <Button
-                        android:id="@+id/button_cancel_launch"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:gravity="center"
-                        android:onClick="onClick"
```

Comment:
why remove all the onClick's ?

Reply:
The onClick handler's are set inside the fragments via databinding. As a result the declaration in the XML files is redundant